src/EventStore.Core/Services/Gossip/GossipServiceBase.cs

```diff
@@ -27,7 +27,7 @@ public abstract class GossipServiceBase : IHandle<SystemMessage.SystemInit>,
 		public const int GossipRoundStartupThreshold = 20;
 		public static readonly TimeSpan DnsRetryTimeout = TimeSpan.FromMilliseconds(1000);
 		public static readonly TimeSpan GossipStartupInterval = TimeSpan.FromMilliseconds(100);
-		private static readonly TimeSpan DeadMemberRemovalTimeout = TimeSpan.FromMinutes(30);
+		private readonly TimeSpan DeadMemberRemovalTimeout = TimeSpan.FromMinutes(30);
```

Comment:
```suggestion
		private readonly TimeSpan _deadMemberRemovalTimeout;
```
No longer needed as we will always be passing this in.